src/check_sentryio_limits.py

```diff
@@ -0,0 +1,123 @@
+p.add_argument('-v', '--verbose', action='store_true', dest='verbose',
+               help='Print detailed stats')
+args = p.parse_args()
+
+
+def get_teams(organization, bearer):
+    '''Return a list of all teams in the account'''
+    teams = []
+    headers = {'Authorization': 'Bearer  {}'.format(bearer)}
+    r = requests.get('https://sentry.io/api/0/organizations/{}/teams/'.format(
+        organization), headers=headers)
+    teams = r.json()
+    return(teams)
+
+
+def get_dsns_from_project(organization, project, bearer):
+    '''Retun a list of DSNs for the project'''
+    dsns = []
+    headers = {'Authorization': 'Bearer {}'.format(bearer)}
+    r = requests.get('https://sentry.io/api/0/projects/{}/{}/keys/'.format(
+        organization, project), headers=headers)
+    dsns = r.json()
+
+    return(dsns)
+
+
+def main():
+
+    teams = get_teams(args.organization, args.bearer)
+
+    '''Filter teams if list is provided'''
+    if args.teams:
+        filtered_teams = []
+        for team in teams:
+            if team['slug'] in args.teams or team['name'] in args.teams:
+                filtered_teams.append(team)
+        teams = filtered_teams
+
+    '''Set exit code and global event counter'''
+    exit = 0
+    events = 0
+
+    '''Iterate over teams, their projects and sum up their keys' rates'''
+    for team in teams:
+        for project in team['projects']:
+            team['summed_events'] = 0
+
+            dsns = get_dsns_from_project(
+                args.organization, project['slug'], args.bearer)
+            for dsn in dsns:
+                if dsn['rateLimit']:
+                    if type(events) is int:
+                        events += int(dsn['rateLimit']['count'] * 60 /
+                                      (dsn['rateLimit']['window']))
+                    if type(team['summed_events']) is int:
+                        team['summed_events'] += int(
+                                dsn['rateLimit']['count'] * 60 /
+                                (dsn['rateLimit']['window']))
+                else:
+                    events = None
+                    team['summed_events'] = None
+
+                if args.verbose:
+                    print('Team: {}, Project: {}, Key: {}, Limit: {}'.format(
+                        team['name'], project['name'], dsn['name'],
+                        dsn['rateLimit']))
+
+            '''Check if this team is over the team limit'''
+            if args.perteamlimit and not team['summed_events']:
+                exit = 1
+                print('WARNING: unlimited events configured for team: {}\
+                    '.format(team['name']))
+            elif args.perteamlimit and team['summed_events'] and \
+                    team['summed_events'] > args.perteamlimit:
+                exit = 1
+                print('WARNING: {} are configure of {} allowed for team: {}\
+                    '.format(team['summed_events'], args.perteamlimit,
```

Comment:
The backslash is not needed:
```suggestion
                print('WARNING: {} are configure of {} allowed for team: {}'
                      .format(team['summed_events'], args.perteamlimit,
```

Reply:
done